src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -233,6 +233,23 @@ pal::string_t get_deps_file(
 }
 
 /**
+* Multilevel Lookup is enabled by default
+*  It can be disabled by setting DOTNET_MULTILEVEL_LOOKUP env var to zero
+*/
+bool multilevel_lookup_enabled()
+{
+    pal::string_t env_lookup;
+    bool multilevel_lookup = true;
+
+    if (pal::getenv(_X("DOTNET_MULTILEVEL_LOOKUP"), &env_lookup))
+    {
+        auto env_val = pal::xtoi(env_lookup.c_str());
+        multilevel_lookup = (env_val != 0);
```

Comment:
I can change that as well. I thought that any nonzero value would be enough to set it.